CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+## Testing
```

Comment:
@chadwhitacre Do we have some guidelines already for `CONTRIBUTING.md`?

Reply:
Looks like we [don't have](https://github.com/getsentry/.github) a global CONTRIBUTING.md that this would override, wfm to start one here. 👍